Update README.md in Web development with examples of Full Stack Technologies #198 issue

dadigavasavipriya opened the pull request:
This pull request introduces comprehensive resource lists for various technology stacks used in web development. These stacks include LAMP, MEAN, Ruby on Rails, Python Django, Java, and ASP.NET. Each section contains curated links to tutorials, guides, and project ideas to help developers master full-stack development with these technologies.

Additions:

1.Introduction to Technology Stacks
Overview of full-stack development and the importance of mastering specific technology stacks.

2.Building Web Applications with LAMP Stack
Resources on Linux, Apache, MySQL, PHP.
Links to tutorials, roadmaps, and project examples.

3.Creating Web Apps with MEAN Stack
Resources on MongoDB, Express, Angular, Node.js.
Links to tutorials, roadmaps, and project examples.

4.Creating Web Applications with Ruby on Rails
Resources on Ruby, Rails, SQLite/MySQL/PostgreSQL, JavaScript.
Links to tutorials, roadmaps, and project examples.

5.Python Web Development using Django
Resources on Python, Django, React.js, MySQL.
Links to tutorials, roadmaps, and project examples.

6.Full-Stack Development with Java
Resources on Java, Spring Boot, React, MongoDB.
Links to tutorials, roadmaps, and project examples.

7.Developing Web Applications with ASP.NET
Resources on ASP.NET, C#, SQL Server.
Links to tutorials, roadmaps, and project examples.

8.Creating Web Apps with MERN Stack
Resources on MongoDB, Express, React, Node.js.
Links to tutorials, roadmaps, and project examples.

jfmartinz commented on Web Development/readme.md:
```diff
@@ -43,6 +43,8 @@ This category is a curated collection of valuable web development resources cont
   - [Beginner level](#beginner-level-projects)
   - [Intermediate level](#intermediate-level-projects)
   - [Advanced level](#advance-level-projects)
+- [Full-stack with various technology stacks](#various-technolgy-stacks)
```
```suggestion
- [Technology Stacks](#technolgy-stacks)
```

jfmartinz commented on Web Development/readme.md:
```diff
@@ -1658,3 +1660,529 @@ This section contains hundreds of coding challenges that test your knowledge of
       </tr>  
   </table>
+
+### Various Technolgy Stacks
```
```suggestion
### Technology Stacks
```

jfmartinz reviewed (changes requested):
Also remove the description in each stacks in the table, only add description into the Technology Stacks and describe the content or about the resources included. Thanks

jfmartinz reviewed (approved):
Please remove these descriptions:
![image](https://github.com/jfmartinz/ResourceHub/assets/129386460/f4c86fb0-bfd4-4457-99da-d380b423f774)
![image](https://github.com/jfmartinz/ResourceHub/assets/129386460/815bdcbc-068c-480d-9e58-0fd35e092dd4)

All the descriptions you added in the table. Only technology stacks should have a description, other tables below, please remove.


Another one is, please add those stacks in the table of contents, the same thing with what you see in the Projects section:

![image](https://github.com/jfmartinz/ResourceHub/assets/129386460/e2057f52-b7f3-4901-ac32-806a47ce1143)

jfmartinz reviewed (approved):
Thanks @dadigavasavipriya

dadigavasavipriya commented:
@jfmartinz Thank you for your feedback. I have removed the descriptions from the tables as requested, except for the Technology Stacks Additionally, I have added the Technology Stacks section to the table of contents, following the same format as the Projects section.
Thank you once again for assigning me this issue. Please let me know if there are any further adjustments needed.

jfmartinz commented:
Merged your PR, thansk for your contribution

dadigavasavipriya commented:
You're welcome, @jfmartinz! If there is any further changes required, please let me know.I'm glad to contribute to the project.
